assembl/views/api/post.py

```diff
@@ -313,16 +373,54 @@ def get_posts(request):
         posts = posts.order_by(Content.body_text_index.score_name.desc())
     else:
         posts = posts.order_by(Content.id)
-    print str(posts)
+    # print str(posts)
 
     no_of_posts = 0
     no_of_posts_viewed_by_user = 0
 
+
+    def post_and_its_whole_line_of_descent_are_deleted(post):
```

Comment:
Add docstrings like:
`
"""
Counts the root post and every post that is it's children, AND their children (therefore, the entire tree of posts under the root post) are deleted or not.
:param: post: The root post to count from
:type: post: assembl.models.Content
:rtype: boolean
"""
`

Very useful to start documenting the back-end as we go along in Sphinx style RST in order to improve overall code documentation. See here for more examples: http://thomas-cokelaer.info/tutorials/sphinx/docstring_python.html